base_multi_company/README.rst

```diff
@@ -0,0 +1,108 @@
+==================
+Multi Company Base
+==================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Production%2FStable-green.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Production/Stable
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fmulti--company-lightgray.png?logo=github
+    :target: https://github.com/OCA/multi-company/tree/14.0/base_multi_company
+    :alt: OCA/multi-company
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-base_multi_company
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/133/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module will provide a way to change the way Odoo manages a 'multi-company'
+implentation.
+
+Abstract
+--------
+
+Odoo traditional implementation of multi-company:
+
+- Some models contain a field named Company (company_id) that allows to set one company or None
+  in order to:
+  - Limit access to that company if set.
+  - not limiting access to any company if not set.
+
+This module breaks that in order to introduce a fine company access.
+e.g.: If you want to give record access to company A and B but not for C.
+
+This module is not doing anything by its own but provide a transversal implementation
+for further ones.
+e.g.: If you want to implement OCA multi-company behaviour for products, install
+also the 'product_multi_company' module.
```

Comment:
You can also mention `partner_multi_company` as the 2 well known modules implementing this.